wakuv2/waku.go

```diff
@@ -1373,21 +1279,19 @@ func (w *Waku) Stop() error {
 	return nil
 }
 
-func (w *Waku) OnNewEnvelopes(envelope *protocol.Envelope, msgType common.MessageType) ([]common.EnvelopeError, error) {
+func (w *Waku) OnNewEnvelopes(envelope *protocol.Envelope, msgType common.MessageType) error {
 	if envelope == nil {
-		return nil, nil
+		return nil
 	}
 
 	recvMessage := common.NewReceivedMessage(envelope, msgType)
 	if recvMessage == nil {
-		return nil, nil
+		return nil
 	}
 
-	envelopeErrors := make([]common.EnvelopeError, 0)
-
 	logger := w.logger.With(zap.String("hash", recvMessage.Hash().Hex()))
 
-	logger.Debug("received new envelope")
+	logger.Info("received new envelope")
```

Comment:
Consider setting this as Debug, as it will add a log for each message received.

Reply:
Agree, yes.